tests/test_mixed_simulator_device.py

```diff
@@ -438,17 +438,17 @@ def test_basis_state_not_at_beginning_error(self, simulator_device_1_wire):
             simulator_device_1_wire.apply([qml.PauliX(0), qml.BasisState(np.array([0]), wires=[0])])
 
     def test_qubit_state_vector_not_at_beginning_error(self, simulator_device_1_wire):
-        """Tests that application of QubitStateVector raises an error if is not
+        """Tests that application of StatePrep raises an error if is not
```

Comment:
Should we still keep the tests for `QubitStateVector` as well until we deprecate it